src/indexer/index.ts

```diff
@@ -90,7 +90,7 @@ export class RunestoneIndexer {
       }
 
       // process blocks that are reorgs
-      for (const blockhash of reorgBlockhashesToIndex) {
+      for (const blockhash of reorgBlockhashesToIndex.toReversed()) {
```

Comment:
Don’t use toReversed, it’s too new of an api call, use reverse instead

Reply:
sounds good, updated